include/CFinalBoneHierarchy.h

```diff
@@ -100,6 +101,35 @@ namespace scene
                 createAnimationKeys(inLevelFixedJoints);
             }
 
+			void fillExportBlob(void* _dataPtr, uint32_t* _size)
```

Comment:
Maybe have a "Blob* SerializeToBlob(void** initialData, uint32_t& initialBlobSize=0);" function for future use in all classes?

With a requirement that outBlob is either NULL or an address of a pointer to memory which can be reallocated using realloc.

And then at the end of the function return *outBlob;

Reply:
any comment?

Reply:
Uh, sorry, must have missed this one. I'll talk to you about this and other issues in few days. As for now I don;t really see the need since `exportAsBlob` functions in mesh writer would do exactly same thing.

Reply:
Blobs are really nice for serialization to a binary struct, which could be used with other data sinks such as other file formats, Networks, MPI, etc.